### ISSUE TITLE
GET /{projectName}/launch/latest returns 500

### ISSUE THREAD

**Issue by ludeknovy** [reportportal/reportportal#458]
Send request:
```
curl -X GET --header 'Accept: application/json' --header 'Authorization: bearer {token}' '{reportPortalUrl}/api/v1/{projectName}/launch/latest'
```

Api returns 500 status code with following error:
```
{
  "error_code": 5000,
  "message": "Unclassified error [Sort must not be null!]"
}
```

Adding `page.sort=desc` to param does not help and still returns the same error.

**Comment by e-olga** [comment]
hi @ludeknovy 
We will add the issue to our backlog.
To get the correct results, please send "page.sort"="start_time,desc" along with project name.
Regards,
Olga

**Comment by evjlobanova** [comment]
Fixed in version 5.0
